trade-finance-manager-api/src/v1/content-strings/index.js

```diff
@@ -294,11 +312,8 @@ const DEAL = {
           textList: [],
         },
         18: {
-          text: 'The fees, interest rate and/or risk margin apply to the whole Cover Period, and have been set in accordance with:',
-          textList: [
-            'The bank’s normal pricing policies',
-            'Include, if any, pricing requirements set by UKEF',
-          ],
+          text: "Any applicable fee-s-, interest rate and/or Risk Margin Fee apply to the whole Cover Period of the Covered Transaction, and have been set in accordance with the Bank's normal pricing policies and include, if any, overall pricing requirements notified by UKEF.",
```

Comment:
Is `fee-s-` intentional here, or is it a typo on the ACs?

Reply:
Have asked, thank you for raising this.